packages/shared/StudioAPI.ts

```diff
@@ -11,7 +11,7 @@ export abstract class StudioAPI {
   abstract getCategories(): Promise<Category[]>;
   abstract getRecipesByCategory(categoryId: string): Promise<Recipe[]>;
   abstract getRecipeById(recipeId: string): Promise<Recipe>;
-  abstract getModelById(modelId: string): Promise<ModelInfo>;
+  abstract getModelById(modelId: string): ModelInfo;
```

Comment:
This needs to be returning a Promise, as the value will be returned by exchanging messags